### PR TITLE
Task to setup private key on satellite server to communicate with libvirt hypervisor

### PR DESCRIPTION
**Pull request by sghai**

### CODE REVIEW

**Comment by sghai** [comment]
I've already generated a key pair and placed it in sesame server. We will wget the private key and place it on all Sat servers via this task, so that each satellite can communicate with hypervisor in a secure way and with same key every time.

**Comment by sghai** [comment]
This will cover the 2) point in https://github.com/SatelliteQE/robottelo/issues/3063

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -479,6 +479,16 @@ def setup_foreman_discovery():
     run('rm -rf {0}'.format(template_file))
 
 
+def setup_libvirt_key():
+    """Task to setup key pairs for secure comminication between Satellite server
+    and libvirt hypervisor."""
+    key_url = os.environ.get('LIBVIRT_KEY_URL')
```
Comment (kbidarkar) [inline automation_tools/__init__.py]: We do have the corresponding entry updated in jenkins configuration file: `Installation Environment Variables`

Comment (kbidarkar) [inline automation_tools/__init__.py]: :+1:

**Comment by kbidarkar** [comment]
ACK

**Comment by elyezer** [comment]
ACK

**Pull request merged**